libcxx/include/__bit/countr.h

```diff
@@ -38,29 +40,45 @@ _LIBCPP_BEGIN_NAMESPACE_STD
   return __builtin_ctzll(__x);
```

Comment:
For a follow-up patch, we should replace uses of `__libcpp_ctz` in the codebase by calls to `__countr_zero` (there's only a few). This will eventually allow us to clean up `__libcpp_ctz` altogether.